src/script/ceph-backport.sh

```diff
@@ -22,7 +22,7 @@ set -e
 
 full_path="$0"
 
-SCRIPT_VERSION="16.0.0.6848"
+SCRIPT_VERSION="17.0.0.7424"
```

Comment:
@smithfarm how do we choose the version number here, I think loicd had a question regarding it in IRC

Reply:
The ceph-backport.sh version number is incremented by running `src/script/ceph-backport.sh --update-version` and including the results of that as a commit in the PR (as I am doing here).

The version number is x.y.z.c where x.y.z is the Ceph version number and c is the number of commits that have been added since the Ceph version number was last updated.

The code for `--update-version` can be viewed in the `update_version_number_and_exit` function of ceph-backport.sh.

While the ceph-backport.sh version number is based on the Ceph version number, there is no effort to keep it up-to-date with the Ceph version number. The only purpose of the ceph-backport.sh version number is to be able to tell how old the ceph-backport.sh code is that someone is running when they report problems.